ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md

```diff
@@ -0,0 +1,245 @@
+
+  > [Info] Further information about the error displayed can be found in the *Logs* tab.
+
+- Click the checkbox to select the action to be checked.    
+The (editing) toolbar is displayed.
+
+  > [Info] If several action have been filtered according to status, it is also possible to select them all simultaneously. Click the checkbox in the column header to select all actions filtered. For detailed information about this process, see [Retry a process action](#to-be-completed).
+
+- Click the [RETRY] button to execute the action again.
+
+
+
+### Next steps
+
+- [Check the token status and content](#check-the-token-status-and-content)
+
+### See also
+
+- [Troubleshooting](#to-be-determined)
+
+## Check the token status and content
+
+The *Tokens* tab displays the tokens that are being or have been processed. A token is a piece of data needed to complete an action, for example a document. When a transaction is performed, a token is generated, for instance an invoice, which in turn initiates the workflow process.
+
+A process is started with one token at the start point, which will be processed in each action executed and, if necessary, turned into several tokens. It is therefore possible that a place contains several tokens, for example, several delivery notes issued for different parts of a same order. Each token will then be processed separately, that means, that the subsequent actions will be executed once for each available token.
+
+### Prerequisites
+
+- A workflow is created, see [Create a workflow](01_ManageWorkflows.md#create-a-workflow).
+- A workflow process has been initiated, see [Link](#to-be-determined).
+
+
+### Procedure
+
+*Workflows > Menu entry Processes > Tab OVERVIEW > Select process > Tab Tokens*
+
+[comment]: <> (Specify Tab Tokens or start with OVERVIEW -> clicking on place shows tokens too?)
+
+![Workflow](/Assets/Screenshots/ActindoWorkFlow/Workflows/Tokens.png "[Workflow]")
+
+1. Click a place in the workflow process diagram to display the available token(s).  
+The available token(s) in the selected place are displayed in the *Tokens* tab.
+
+  > [Info] If the workflow process is finished, all available tokens are placed in the end point.
+
+2.  Check the status of the selected token(s) in the *Status* column.  
+  The following statuses are available:
+
+  - Processed (green)
+  - Being processed (blue)
+  - Ready (grey)
+  - Error (red)
+
+
+3. Click the *Hide processed* ![Toggle](/Assets/Icons/Toggle.png "[Toggle]") (Toggle) button to hide the processed tokens.    
+All unprocessed tokens are displayed.  
+
+> [Info] Tokens can also be filtered with the filter options. Click the ![Filter](/Assets/Icons/Filter.png "[Filter]") (Filter) button to display the filter options. For detailed information, see [Link](#to-be-completed).
+
+4. Check the place where the token(s) are located in the *Place* column.  
+
+5. Check what type of data is the selected token in the *Data* column.
+
+
```

Comment:
möglicher ablauf dieser procedure:

- tab tokens wählen
- toggle hide processed deaktivieren (um alle tokens anzuzeigen)
- tokens/places in diagramm anschauen (status farben bei places, schwarze quadrate an places, token status in actions erklären)
- place in diagramm auswählen (je nach status wird token in tab angezeigt)
- status und weitere infos zu token in tab tokens anschauen (spalten erklären: status, place, data, next possible action(s)
- verweis auf actions
- ggf. logs einschließen -> mit oli klären (vielleicht auch nur in troubleshooting oder in separatem kapitel)